### DISCUSSION TITLE
Is it possible to make a simple 0 1 ASCII PBM

### DISCUSSION THREAD

**Discussion by Lg0enga** [dlemstra/Magick.NET#1586] [Help] [answered]
In the ImageMagick [documentation ](https://imagemagick.org/Usage/formats/#pbmplus) you will find that a simple version of a PBM file can be created as ASCII PBM. This only contains 0 and 1.

Is it possible to also create an image in this format with Magick.NET?

I want to print the image on a label printer that only supports a byte array with 0 for white and 1 for black.

Thank you in advance!

**Comment by dlemstra** [comment, marked as answer]
This should be possible by writing the image to a file name `YourName.pbm` Is that not working?

**Comment by Lg0enga** [comment]
We succeeded with the .pbm

I was already using this extension, but got an offset when printing. This was due to the fact that the printer could not handle the dimensions that were first in the byte array. I took this out for the print.

**Comment by dlemstra** [comment]
Thanks for coming back and explaining how you fixed it.
